aten/src/TH/generic/THStorage.cpp

```diff
@@ -21,18 +21,29 @@ size_t THStorage_(elementSize)()
 
 THStorage* THStorage_(new)(void)
 {
-  return THStorage_new(at::CTypeToScalarType<th::from_type<real>>::to());
```

Comment:
Wait, this seems worse than before. We're trying to get rid of the `generic` folder, but the implementations migrated back into `generic`?

Reply:
Oh, nevermind, it's because you're using the constructor now.